Makefile

```diff
@@ -20,4 +20,5 @@ protocol:
 
 #Build and run the unit tests
 test:
-	./travis.sh $(IDEA_VERSION)
+	./travis.sh $(IDEA_VERSION) $(ANT_TARGET)
```

Comment:
Pass the target through to ant.

Just to remind you how this works:
  .travis.yml kicks off make, which kicks off travis.sh, which downloads the IDEA version and then kicks off ant.